### DISCUSSION TITLE
Change of port for Web OTBR

### DISCUSSION THREAD

**Discussion by Koky05** [b2un0/silabs-multipan-docker#2] [Q&A] [answered]
Hi,
I use docker-compose file to control all my docker containers but I also use InfluxDB for long therm statistics. Is it possible to change port 8086 for OTBR because it collide with InfluxDB?

**Comment by Mermarky** [comment, marked as answer]
Not an expert but I would expect you need to change OTBR_WEB_PORT: "8086" to another port

**Comment by Koky05** [comment]
Yes, that was it. Thanks.
